ddtrace/contrib/logging/patch.py

```diff
@@ -4,17 +4,20 @@
 
 import ddtrace
 
+from . import constants
 from ...utils.wrappers import unwrap as _u
 from ...vendor.wrapt import wrap_function_wrapper as _w
 
 
-RECORD_ATTR_TRACE_ID = "dd.trace_id"
-RECORD_ATTR_SPAN_ID = "dd.span_id"
-RECORD_ATTR_ENV = "dd.env"
-RECORD_ATTR_VERSION = "dd.version"
-RECORD_ATTR_SERVICE = "dd.service"
-RECORD_ATTR_VALUE_ZERO = "0"
-RECORD_ATTR_VALUE_EMPTY = ""
+# Deprecated and replaced by constants module
+RECORD_ATTR_TRACE_ID = constants.RECORD_ATTR_TRACE_ID
+RECORD_ATTR_SPAN_ID = constants.RECORD_ATTR_SPAN_ID
+RECORD_ATTR_ENV = constants.RECORD_ATTR_ENV
+RECORD_ATTR_VERSION = constants.RECORD_ATTR_VERSION
+RECORD_ATTR_SERVICE = constants.RECORD_ATTR_SERVICE
+RECORD_ATTR_VALUE_ZERO = constants.RECORD_ATTR_VALUE_ZERO
+RECORD_ATTR_VALUE_EMPTY = constants.RECORD_ATTR_VALUE_EMPTY
+
```

Comment:
Would a similar approach to https://github.com/DataDog/dd-trace-py/blob/ed486ba62cdcc8b8eecf5f2177fa0ab5e105b81c/ddtrace/constants.py with the `__getattr__` module method be of value here to add a deprecation warning?

Reply:
TIL `__getattr__` on a module!

That's what I was looking for but didn't know how to accomplish. Will update!